### PR TITLE
More clean-ups and fixes of baseline comparing Mojos

### PR DESCRIPTION
**Pull request by HannesWell**
After https://github.com/eclipse-tycho/tycho/pull/1327 has been merged, the results of https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/pull/564 are not as expected.
This PR adds some debug output for testing and tries to build the platform with it.

### CODE REVIEW

File: .github/workflows/verify-platform.yml [hunk 1]
```diff
@@ -24,7 +24,8 @@ jobs:
        path: 'platform'
        submodules: 'recursive'
        lfs: true
-       repository: 'eclipse-platform/eclipse.platform.releng.aggregator'
+       repository: 'HannesWell/eclipse.platform.releng.aggregator'
```
Comment (laeubi) [inline .github/workflows/verify-platform.yml:27]: Is this intended?

File: .github/workflows/verify-platform.yml [hunk 2]
```diff
@@ -61,7 +62,8 @@ jobs:
        JAVA_HOME: ${{ env.JAVA_HOME_17_X64 }}
       working-directory: 'platform'
       run: >- 
-        mvn -U -V -e -B -ntp
+        mvn -U -V -e -B -ntp -X
```
Comment (laeubi) [inline .github/workflows/verify-platform.yml:65]: Will this be reverted or is this by intention? The verification here should really not fail for any baseline comparison as we never change any code.

File: .github/workflows/verify-platform.yml (hunk 1 above)
Comment (HannesWell) [reply .github/workflows/verify-platform.yml:27]: It is an intended change for testing this PR, but I will revert that before merging this.

File: .github/workflows/verify-platform.yml (hunk 2 above)
Comment (HannesWell) [reply .github/workflows/verify-platform.yml:65]: Same as above.

**Comment by HannesWell** [comment]
There was indeed a regression in #1327.
In contrast to `FileUtils.copyStreamToFile()` the method `Files.copy()` fails if the target file already exists, unless REPLACE_EXISTING is passed. And `Files.createTempFile()` returns the path of an existing file. So without passing `REPLACE_EXISTING` the comparison of nested zip cannot work.

**Comment by HannesWell** [comment]
Besides that I think if the zip comparison failed with an exception a warning should always be printed displaying the exception and not only a simple delta without context should be presented.
Without that it is hard to tell why there is a difference and an IOexception is actually always not acceptable.

File: .github/workflows/verify-platform.yml (hunk 1 above)
Comment (HannesWell) [reply .github/workflows/verify-platform.yml]: > It is an intended change for testing this PR, but I will revert that before merging this.

Done.

**Comment by mickaelistria** [comment]
> the comparison of nested zip cannot work.

I believe Eclipse Platform build is badly affected, see https://ci.eclipse.org/platform/job/eclipse.platform/job/PR-183/2/console . Can we have a remediation ASAP?

**Comment by HannesWell** [comment]
> > the comparison of nested zip cannot work.
> 
> I believe Eclipse Platform build is badly affected, see https://ci.eclipse.org/platform/job/eclipse.platform/job/PR-183/2/console . Can we have a remediation ASAP?

Yep merged it, which was the plan anyway. :)

**Comment by mickaelistria** [comment]
thank you!

**Comment by laeubi** [comment]
@akurtakov @mickaelistria given all these "can we get ASAP" demands, I really like to ask if we can get at least more **COMPUTER POWER** resources for Tycho because that currently stalls our development a lot!

I have opened https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/1955

But also @merks I really would appreciate to take this into account for any "funding" discussions, its really a pitty that we are so limited here and have static assignments of build nodes really seems ancient given today "cloud computing" capabilities.

I have even sponsored an own build node already https://ci.eclipse.org/tycho/computer/laeubisoft%2Dbuild%2Dnode/ but still if a lot of changes "rush in" that's not enough and feels a bit odd that one has to contribute development AND processing power. But without that, we have had a backlogs that take half a day to be processed in the past, this does not fit the "we need fixes ASAP from Tycho" demands...

**Pull request merged**